source/ncengine/graphics2/diligent/resource/GlobalResourceSignature.h

```diff
@@ -11,44 +14,46 @@
 
 namespace nc::graphics
 {
-class GlobalTextureBufferResource;
-class GlobalEnvironmentResource;
+class TextureBufferResource;
+class EnvironmentBufferResource;
 
-class GlobalResourceSignature
+class PerFrameResourceSignature
 {
     public:
-        explicit GlobalResourceSignature(Diligent::IDeviceContext& context,
+        explicit PerFrameResourceSignature(Diligent::IDeviceContext& context,
```

Comment:
I'm thinking you meant to delete this file. There is a `PerFrameResourceSignature` header/source that looks p. similar.